submission-folder/Anton.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * HOMEWORK
+ * rename this file to *your name*.ts
+ * Translate this JS code into valid TypeScript that follows good practices,
+ * following instructions in the comments
+ */
+
+// replace all `var` declarations appropriatly
+
+const NINE_THOUSAND = 9000
+let number = 6
+number = 7
+
+// create a type alias (rmb type alias names use PascalCase)
+type Student = {
+    name : string
+    year : number
+    friends : string[]
+    favouriteVariable : number | string
+}
+let bob : Student
+let steve : Student
+let mary : Student
+
+bob = {
+  name: 'Bob',
+  year: 2,
+  friends: ['Steve'],
+  favouriteVariable: 23,
+}
+
+steve = {
+  name: 'Steve',
+  year: 1,
+  friends: ['Bob', 'Mary'],
+  favouriteVariable: 'hello', // note that this shall ONLY be string or number
+}
+
+mary = {
+  name: 'Mary',
+  year: 1,
+  friends: ['Steve'],
+  favouriteVariable: 100,
+}
+type StudentArray = Student[]
+let students : StudentArray = [bob, steve, mary] // assign an appropriate type to `students`
+
+// convert to anonymous function
+// use string literals
+// add appropriate type declarations to function args
+const printNameAndYear = (student : Student) => {
+    console.log(student.name + ' is year ' + student.year)
```

Comment:
you are suppose to use replace the string concatenation operations here by using string literals